### DISCUSSION TITLE
Bug with diff rendering when a UTF-8 BOM has been added

### DISCUSSION THREAD

**Discussion by ForNeVeR** [community/community#7593] [Pull Requests]
Today, I've received a PR from a contributor who has decided to add a UTF-8 BOM to a file: https://github.com/ForNeVeR/Fenrir/pull/60/files

This diff renders strangely on my computer:
![image](https://user-images.githubusercontent.com/92793/141325638-5799ea88-2b7c-4077-84ce-b320e2199d89.png)

As if there was a line 0 which got deleted from the repo, while it actually isn't. The "line 0" is actually the last line of the same file.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
